lib/kd-controller.js

```diff
@@ -57,11 +57,6 @@ export default class KDController extends kd.Object {
   }
 
   _err (e) {
-    if (this.modalPanel) {
-      this.modalPanel.destroy()
-      this.modalPanel = null
```

Comment:
Those are using their own instances, `modalPanel` itself is never used and never being assigned to any value.